server/server.go

```diff
@@ -39,10 +37,7 @@ type Server struct {
 // New instantiates a new Server.
 // Callers must then invoke Start to run the Server.
 func New(c *Config) (*Server, error) {
-	logger, err := configureLogger(c.Logging)
-	if err != nil {
-		return nil, errors.Wrap(err, "failed to initialize logging")
-	}
+	logger := baseapp.NewLogger(c.Logging)
```

Comment:
Per internal discussion, let's add `Level` to the existing config and shim it here:

```golang
logger := baseapp.NewLogger(baseapp.LoggingConfig{
  Level:  c.Logging.Level,
  Pretty: c.Logging.Text,
})
```